.eslintrc

```diff
@@ -1,15 +1,12 @@
 {
   "parser": "@typescript-eslint/parser",
   "extends": [
-    "standard",
-    "standard-react",
```

Comment:
we had to choose between standard and airbnb.

I think we should go with airbnb. We use it in hops too.